lib/devise/uncommon_password/model.rb

```diff
@@ -26,7 +26,11 @@ def self.common_passwords
 
       def not_common_password
         if Devise::Models::UncommonPassword.common_passwords.include? password.downcase
-          errors.add(:password, "is a very common password. Please choose something harder to guess.")
+          errors.add(:password, def not_common_password
+            if Devise::Models::UncommonPassword.common_passwords.include? password.downcase
+              errors.add(:password, I18n.t('errors.messages.common_password'))
```

Comment:
This string doesnt exist yet in our app

Reply:
What I mean is, we need to add it. That is why you're not seeing the error message when testing.

Reply:
okay, thank you!